<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kleo::Dialogs::ExpiryDialog</class>
 <widget class="QWidget" name="Kleo::Dialogs::ExpiryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Certificate Date of Expiry</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please select when to expire this certificate:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="neverRB">
     <property name="text">
      <string>Never</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QRadioButton" name="inRB">
       <property name="text">
        <string>In</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="inSB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="inCB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Days</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Weeks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Months</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Years</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="onRB">
     <property name="text">
      <string>On this day:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCalendarWidget" name="onCW">
       <property name="gridVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>418</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>onRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>onCW</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>inCB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>inSB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
